recipe/meta.yaml

```diff
@@ -29,13 +29,13 @@ requirements:
     - ipopt
     - setuptools
     - cython
-    - numpy x.x
+    - numpy 1.8.*
```

Comment:
Only numpy `1.11` is available across all platforms and python versions in conda-forge.

Reply:
This only builds for macosx and linux. From the docs it seemed like this should work.

Reply:
Yes, but scipy is built against numpy 1.9, so you need at least 1.9 here